src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static User firstUserOk;
+    private static User secondUserOk;
+    private static User thirdUserOk;
+    private static User userPasswordNotOk;
+    private static User userAgeNotOk;
+    private static User userNameNullNotOk;
+
+    @BeforeClass
+    public static void beforeClass() {
+        registrationService = new RegistrationServiceImpl();
+        firstUserOk = setUserData(1111L, "Bob", "qwerty", 18);
+        secondUserOk = setUserData(2222L, "Alice", "asdfghjk", 29);
+        thirdUserOk = setUserData(3333L, "Jhon", "zxcvbnm", 25);
+        userPasswordNotOk = setUserData(-1111L, "BadBob", "qwer", 19);
+        userAgeNotOk = setUserData(-2222L, "BadAlice", "asdfghjk", 17);
+        userNameNullNotOk = setUserData(-2222L, null, "asdfghjk", 23);
+    }
+
+    @Before
+    public void setUp() {
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_nullName_notOk() {
```

Comment:
let's rename methods register_nullLogin_notOk(), because this is more understandable)